templates/header.hbs

```diff
@@ -1,19 +1,24 @@
 <header class="header">
-  <div class="logo">
-    {{#link 'help_center'}}
-      <img src="{{settings.logo}}" alt="{{t 'home_page' name=help_center.name}}">
-    {{/link}}
-  </div>
-  <div class="nav-wrapper">
-    <button class="menu-button" aria-controls="user-nav" aria-expanded="false" aria-label="{{t 'toggle_navigation'}}">
-      <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" focusable="false" viewBox="0 0 16 16" class="icon-menu">
-        <path fill="none" stroke="currentColor" stroke-linecap="round" d="M1.5 3.5h13m-13 4h13m-13 4h13"/>
-      </svg>
-    </button>
-    <nav class="user-nav" id="user-nav">
-      {{link 'community'}}
-      {{link 'new_request' class='submit-a-request'}}
-    </nav>
-    {{user_info}}
+  <div class="header__inner">
```

Comment:
opinion question: Is it better to use the mixin `@include max-width-container;` on every element we need centered, or is it better to have a `.conatiner` type element that is reused?

Reply:
The max-width-container mixin handles a media query and other stuff, too. It's a built-in with ZenDesk. I'm reluctant to muck with it too much because then pages that I intend to only lightly touch might require more refactoring.

Reply:
Oh! There is a `.container` class in `_base.scss` online 109. It uses the max-width-container mixin. XD